mixins/bits/templates/bits-service-account.yml

```diff
@@ -6,6 +6,12 @@ kind: ServiceAccount
 metadata:
   name: bits-service
   namespace: {{ .Release.Namespace }}
+{{- if gt (len .Values.kube.image_pull_secrets) 0 }}
+imagePullSecrets:
+  {{- range $secret_name := $.Values.kube.image_pull_secrets }}
+  - name: "{{ $secret_name }}"
+  {{- end }}
+{{- end }}
```

Comment:
This block of code is repeated often enough that you should define a template in `chart/template/_helpers.tpl` and use that in all places. Look at `component.labels` in that file for an example.

I can't think of a good name right now, so I would call it `kubecf.imagePullSecrets` unless you can think of something better.

Reply:
confirmed